### ISSUE TITLE
MISP keys not present

### ISSUE THREAD

**Issue by BeRambo** [CIRCL/AIL-framework#491]
Hi everyone

I'm having a strange problem again. I'm trying to send events to MISP. Yesterday the MISP_TheHive_feeder.py script still found my MISP keys. Today I get the message they are not present.

- I've replaced the script with the one from the master branch
- Tried running with python3
- Tried restarting ofcourse
- Reinstalled all requirements
- Tried both http and https in misp_url
- I'm running the latest version of AIL on Ubuntu 18.04 LTS
- Tried chmod 777 on mispKEYS.py
- Reinstall pymisp with pip3

```
(AILENV) bramv@ail:~/AIL-framework$ python3 bin/MISP_The_Hive_feeder.py
Misp keys not present
The HIVE keys not present
^CTraceback (most recent call last):
  File "bin/MISP_The_Hive_feeder.py", line 224, in <module>
    time.sleep(1)
KeyboardInterrupt
```

mispKEYS.py:
```
#!/usr/bin/env python
# -*- coding: utf-8 -*-

misp_url = 'https://misp.*censored*.com'
misp_key = '*censored*' # The MISP auth key can be found on the MISP web interface under the automation section
misp_verifycert = False #Because curl doesn't work with https

```

**Comment by annetteshajan** [comment]
Yes same issue @Terrtia 
Any idea on how to resolve?

**Comment by Terrtia** [comment]
Hi @BeRambo , @annetteshajan !

Fixed with 762b517150ac46898331e52878496752681be167,
However, there is currently an issue on MISP side, all the tags are currently missing.
This issue should be soon resolved https://github.com/MISP/MISP/issues/6032.

**Comment by Terrtia** [comment]
Fixed.
